### PR TITLE
add an alias endpoint with a type parameter

### PR DESCRIPTION
**Pull request by jum-s**

### CODE REVIEW

**Review by maxlath — DISMISSED** [review]

**Comment by jum-s** [comment]
linked to client branch [#218](https://github.com/inventaire/inventaire-client/pull/218/files)

File: server/data/aliases.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  type: {
+    whitelist: typesNames
+  }
+}
+
+module.exports = (req, res) => {
+  sanitize(req, res, sanitization)
+  .then(params => {
+    const { type } = params
+    return res.json(typesAliases[type])
```
[the remark this answers is not in the document]
Comment (jum-s) [reply server/data/aliases.js]: renamed and wrapped in `018874b`

**Review by maxlath — APPROVED** [review]

**Comment by maxlath** [comment]
@jum-s ping for review after I pushed some commits (I apparently can't request your review back after you asked for mine)

File: server/controllers/entities/lib/properties/properties_config_builders.js [hunk 2]
```diff
@@ -41,5 +43,16 @@ module.exports = {
         return regexPerType[entityType].test(value)
       }
     })
+  },
+
+  shortlistedEntityValues: property => {
```
Comment (jum-s) [inline server/controllers/entities/lib/properties/properties_config_builders.js:48]: `shortlistedEntityValues` -> `allowedPropertyValues` ?

**Comment by jum-s** [comment]
@maxlath ping after ping for review

**Comment by maxlath** [comment]
@jum-s continuing on the renaming: what about replacing occurrences of `propertiesValuesPerTypesLists` by `allowedValuesPerPropertyAndType`?

**Comment by jum-s** [comment]
`allowedValuesPerTypePerProperty` since `And` can feel horizontal while its a deep object ?

**Comment by maxlath** [comment]
would be `allowedValuesPerPropertyPerType` then, going from the values, up to the object root, but `allowedValuesPerPropertyAndType` sounds more natural to me

**Comment by jum-s** [comment]
of course, my bad

**Pull request merged**